pengine/allocate.c

```diff
@@ -2477,6 +2478,16 @@ create_notifications(resource_t * rsc, notify_data_t * n_data, pe_working_set_t
             for (; gIter != NULL; gIter = gIter->next) {
                 node_t *current_node = (node_t *) gIter->data;
 
+                /* if this stop action is a pseudo action as a result of the current
+                 * node being fenced, this stop action is implied by the fencing 
+                 * action. There's no reason to send the fenced node a stop notification */ 
+                if (stop &&
```

Comment:
Its more than that, if the resource is stopped on that node then there is nothing to send a notification to